clusterware-customize/lib/functions/job-queue.functions.sh

```diff
@@ -268,7 +307,7 @@ job_queue_list_jobs_in_queue() {
     job_status="$2"
 
     job_queue_s3cmd_setup
-    s3_prefix=$(job_queue_bucket_path "${queue}" "${job_status}"/ )
+    s3_prefix=$(job_queue_bucket_put_path "${queue}" "${job_status}"/ )
```

Comment:
Should this be `pull_path` rather than `put_path`? i.e. list jobs in the queue for the node on which you are running the command rather than list jobs for the master node?